File: fizzbuzz/c/domielias/FizzBuzz.c
//Contributor: Domielias
#include <stdio.h>
void isFizzBuzz(int number);

int main(){
  for(int number=1;number<101;number++)
      isFizzBuzz(number);
}
void isFizzBuzz(int number){
  if(number%3==0 && number%5==0)
    printf("%d FizzBuzz\n",number);
  else if(number%3==0)
    printf("%d Fizz\n",number);
  else if(number%5==0)
    printf("%d Buzz\n",number);
}

File: fizzbuzz/lua/MaximumGeeker/fizzbuzz.lua
--[[ FizzBuzz, Lua 5.1 ]]--

local ans = {
  'Fizz',
  'Buzz',
  'FizzBuzz'
}

for i = 1, 100 do
  local c = 0

  if i%3 == 0 then
    c = c + 1
  end

  if i%5 == 0 then
    c = c + 2
  end

  if ans[c] then
    print( i, ans[c] )
  end
end
